internal/service/dynamodb/table_replica_test.go

```diff
@@ -759,3 +795,61 @@ resource "aws_dynamodb_table_replica" "test" {
 }
 `, rName, key))
 }
+
+func testAccTableReplicaConfig_enable_deletion_protection(rName string) string {
+	return acctest.ConfigCompose(
+		acctest.ConfigMultipleRegionProvider(3),
+		fmt.Sprintf(`
+resource "aws_dynamodb_table" "test" {
+  name             = %[1]q
+  hash_key         = "TestTableHashKey"
+  billing_mode     = "PAY_PER_REQUEST"
+  stream_enabled   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+
+  attribute {
+    name = "TestTableHashKey"
+    type = "S"
+  }
+
+  lifecycle {
+    ignore_changes = [replica]
+  }
+}
+
+resource "aws_dynamodb_table_replica" "test" {
+  provider                    = "awsalternate"
+  global_table_arn            = aws_dynamodb_table.test.arn
+  deletion_protection_enabled = true
+}
+`, rName))
+}
+
+func testAccTableReplicaConfig_disable_deletion_protection(rName string) string {
```

Comment:
Would it be possible to combine this and the `testAccTableReplicaConfig_enable_deletion_protection` function and use a boolean to enable/disable deletion protection? It looks like both configurations are very similar and the only difference is setting the `deletion_protection` argument.

Reply:
added